doc/sphinx/proof-engine/ltac.rst

```diff
@@ -1189,12 +1200,15 @@ Pattern matching on terms: match
       to a tactic, that value is the result of the match expression.
 
       If :n:`@ltac_expr` is a tactic with backtracking points, then subsequent
-      failures (after the `*match*` construct) can cause this tactic to backtrack.
+      failures after a :tacn:`lazymatch` or :tacn:`multimatch` (but not :tacn:`match`) can cause
+      backtracking into :n:`@ltac_expr` to select its next success.
+      (:tacn:`match` :n:`…` is equivalent to :tacn:`once` :tacn:`multimatch` `…`.
+      The :tacn:`once` prevents backtracking into the :tacn:`match` after it has succeeded.)
 
    .. note::
       Each |Ltac| construct is processed in two phases: an evaluation phase and an execution phase.
       In most cases, tactics that may change the proof state are applied in the second phase.
-      (Constructs that generate integer, string or syntactic values such as :tacn:`fresh`
+      (Tactics that generate integer, string or syntactic values
```

Comment:
I liked the "constructs" language better, but feel free to use whatever you think is best
```suggestion
      (Constructs that generate integer, string, or syntactic values, such as :tacn:`fresh`,
```

Reply:
We are describing `fresh` et al as tactics.  Not certain we'll call compound tactics like `+`, `||, etc tactics for the longer term.  In this case, I think "tactic" is better because the previous sentence uses "tactic".  It avoids the reader wondering whether "tactic" and "construct" mean the same thing.  I've "construct" in some places, sometimes for variety.